drivers/mtd/filemtd.c

```diff
@@ -449,6 +449,15 @@ static int filemtd_ioctl(FAR struct mtd_dev_s *dev, int cmd,
         }
         break;
 
+      case MTDIOC_ERASESTATE:
+        {
+          uint8_t *result = (uint8_t *)arg;
```

Comment:
should we add FAR for all pointer at least in the common code?

Reply:
You're right, I missed that. I've also added to chip-specific drivers, since they all follow the pattern of adding FAR to every pointer to data.